packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -499,7 +503,10 @@ export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget
                     }
                 };
 
-                const disposable = this.term.onData(sendData);
+                const disposable = new DisposableCollection;
```

Comment:
I am surprised our linter didn't flag this `new` call without parenthesis?

Please add the parenthesis for the sake of consistency with the rest of the code base, thanks :)